docs/releases/2.2.rst

```diff
@@ -0,0 +1,16 @@
+2.2 Update
+==========
+
+
+Added Acct Support for Exec Modules
+===================================
+We updated the exec modules to pass context to get_cleint.
+We  also  updated the state modules to pass context to exec modules.
+In addtion to that, we have  updated the utiles to recive context on get_client and pass to determine_auth
```

Comment:
rogue extra space. spell check. enclose function calls in single "back ticks" in order to designate them as code

Reply:
words `get_client` words